client/src/routes/App/FileTransfer/FileTransfer.js

```diff
@@ -22,6 +21,35 @@ import roomsDispatch from '../../../reducers/rooms';
 
 import './FileTransfer.scss';
 
+
+// adapted from https://github.com/sindresorhus/multi-download/blob/v4.0.0/index.js
+// to take File https://developer.mozilla.org/en-US/docs/Web/API/File
+const delay = milliseconds => new Promise(resolve => {
+  setTimeout(resolve, milliseconds);
+});
+const download = async (file) => {
+  const a = document.createElement('a');
+  a.download = file.name;
+  a.href = URL.createObjectURL(file);
+  a.style.display = 'none';
+  document.body.append(a);
+  a.click();
+
+  // Chrome requires the timeout
+  await delay(100);
+  a.remove();
+};
```

Comment:
Can we use `downloadjs` for this? From what I can see, delay of 1s is something that is to be added for fixing the issue. The individual file download seems to use same process as downloadjs

Reply:
in addition to the 1s delay, to me it looks like this additional 100ms delay before removing the `a` again is not present in downloadjs. it looks like they immediately remove it after clicking: https://github.com/rndme/download/blob/09d6492f47ef18feca39c3d748960dce44f93a89/download.js#L116-L117

maybe chrome introduced the need for this additional short delay somewhere in the last 8 years (the latest release of downloadjs)?

Reply:
I'd say this is a nice elimination of some legacy/unmaintained code, in exchange for a `download` function that's only 10 lines without dependencies

Reply:
Ah I see the timeout. Changes seems fine to me, I'll also test it on my end in couple of browsers to ensure that it doesn't break existing flows.

Reply:
OK! we might be able to reduce the 1000ms, I blamed it to a commit related to cross-origin URLs